### ISSUE TITLE
Can you add a bird 's-eye view to the bottom right corner of the view?

### ISSUE THREAD

**Issue by libaineu2004** [paceholder/nodeeditor#352]
Can you add a bird 's-eye view to the bottom right corner of the view?
For example, like this picture:
![7d6f67032b3271340e3bcd5733eb3e93](https://user-images.githubusercontent.com/11898277/208293812-d14a64fa-d2a6-428f-804b-fd0f9ea16813.png)

**Comment by paceholder** [comment]
Theoretically it must be possible by adding a mini-view-widget to the corner and connecting it to the same scene/graph-model. I'll see what I can do

**Comment by libaineu2004** [comment]
> Theoretically it must be possible by adding a mini-view-widget to the corner and connecting it to the same scene/graph-model. I'll see what I can do

Yes, add an aerial view in the lower right corner. I saw from the website: 
http://cneben.github.io/QuickQanava/index.html
I think this feature is very useful

**Comment by libaineu2004** [comment]
Is there a time plan to implement this function?
